packages/eslint-plugin-gestalt/README.md

```diff
@@ -0,0 +1,89 @@
+# eslint-plugin-gestalt
```

Comment:
@jennyscript love this README - should we also include something on our docs site about this package?

Reply:
I added a new docs section "Eslint Plugin" with similar info to the readme